File: r/encrypt.r
library(crypto)

key <- "A1B2C3D4E5F6H7I8"
data <- readLines("../json.txt")
encrypted <- encrypt(data, key)
print(paste("Encrypted:", encrypted))

encrypt <- function(data, key) {
    aes <- aes(key = key, mode = "ecb", padding = "pkcs7")
    enc <- encrypt(data, aes)
    enc <- base64enc(enc)
    return(enc)
}

# Install
# Rscript -e "install.packages('crypto', repos='http://cran.us.r-project.org')"
# Run
# Rscript encrypt.r

File: dart/encrypt.dart
import 'dart:convert';
import 'dart:io';
import 'package:crypto/crypto.dart';
import 'package:encrypt/encrypt.dart';

void main() {
  final key = 'A1B2C3D4E5F6H7I8';
  final data = File('json.txt').readAsStringSync();
  final encrypted = encrypt(data, key);
  print('Encrypted: $encrypted');
}

String encrypt(String data, String key) {
  final keySha256 = sha256.convert(utf8.encode(key)).bytes;
  final keySha256_16 = keySha256.sublist(0, 16);
  final keySha256_16_base64 = base64.encode(keySha256_16);

  final iv = IV.fromLength(16);
  final encrypter =
      Encrypter(AES(Key.fromBase64(keySha256_16_base64), mode: AESMode.ecb));
  final encrypted = encrypter.encrypt(data, iv: iv);
  return encrypted.base64;
}

// Install
// pub get
// Run
// dart bin/encrypt.dart